tests/baselines/reference/assertionTypePredicates1.errors.txt

```diff
@@ -184,7 +184,7 @@ tests/cases/conformance/controlFlow/assertionTypePredicates1.ts(149,5): error TS
         assert(typeof x === "string");  // Error
         ~~~~~~
 !!! error TS2775: Assertions require every name in the call target to be declared with an explicit type annotation.
-!!! related TS2728 tests/cases/conformance/controlFlow/assertionTypePredicates1.ts:144:11: 'assert' is declared here.
+!!! related TS2782 tests/cases/conformance/controlFlow/assertionTypePredicates1.ts:144:11: 'assert' needs an explicit type annotation.

```

Comment:
as far as I understand, this error was previously incorrect and is now still incorrect. the real problem is that we can't use `asserts` with arrow functions, right?

Reply:
No, the arrow function is fully annotated, but the assert declaration itself needs to be annotated. It's very stupid.

```diff
- const assert = (value: unknown): asserts value => { }
+ const assert: (value: unknown) => asserts value = (value: unknown): asserts value => { }
```

Reply:
Or "better"

```diff
- const assert = (value: unknown): asserts value => { }
+ const assert: (value: unknown) => asserts value = value => { }